src/platform/serviceRegistry.web.ts

```diff
@@ -50,6 +54,15 @@ export function registerTypes(serviceManager: IServiceManager) {
         KernelProgressReporter
     );
 
+    serviceManager.addSingleton<IInterpreterPackages>(IInterpreterPackages, InterpreterPackages);
+    serviceManager.addSingleton<IExtensionSyncActivationService>(
```

Comment:
These things that have to do with interpeters is now in interpeters folder and no longer in telemetry folder.